plat/kvm/x86/setup.c

```diff
@@ -194,7 +194,10 @@ static int paging_init(void)
 
 		rc = ukplat_page_map(&kernel_pt, vaddr, paddr,
 				     len >> PAGE_SHIFT, prot, 0);
-		if (unlikely(rc))
+		/* Unmappings are currently not performed on the low-mem.
```

Comment:
IIRC this is why the multiboot.c code drops memory regions below 1MiB. You can do so as well.

Reply:
Firecracker places the command line below 1MiB.